core/domain/story_services.py

```diff
@@ -304,6 +304,17 @@ def validate_explorations_for_story(exp_ids, raise_error):
                         raise utils.ValidationError(error_string)
                     validation_error_messages.append(error_string)
 
+                if state.interaction.id == 'EndExploration':
+                    recommended_exploration_ids = state.interaction.customization_args['recommendedExplorationIds']['value'] # pylint: disable=line-too-long
```

Comment:
Can't you add a new line at the square bracket? If so, don't use lint pragma here.

Reply:
Ah, yes, done. Thanks!